<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Avoid</class>
 <widget class="QTabWidget" name="Avoid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>运动避障</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="tabCtrl">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="title">
    <string>基本控制</string>
   </attribute>
   <widget class="QPushButton" name="buttonQuit1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleCtrl">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 机器人运动控制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUp">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>向前</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDown">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string> 向后</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLeft">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>向左</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRight">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>向右</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSpin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>旋转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTrack">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>191</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>目标跟踪</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonVoiceTrack">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>191</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>声音跟随</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonInit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>191</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tabSet">
   <attribute name="title">
    <string>参数设置</string>
   </attribute>
   <widget class="QPushButton" name="buttonQuit2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleSet">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 机器人运动参数设置</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSet">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>参数设置</string>
    </property>
    <widget class="QSpinBox" name="spinBoxLeftSpeed">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelLeftSpeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>左轮速度 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRightSpeed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>右轮速度 :</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRightSpeed">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSetSpeed">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>75</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>写入速度</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDirectionAngle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>方向角    :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSetDirectionAngle">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>75</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>写入速度</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSetSafeDis">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>75</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>写入速度</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxDirectionAngle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxSafeDis">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSafeDis">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>安全距离 :</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDownload">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>程序下载</string>
    </property>
    <widget class="QPushButton" name="buttonLoad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>75</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>读取文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDownload">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>75</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonEreasure">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>75</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>擦除</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLoadFile">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>烧写文件</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLoadFilePath">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>请读取烧写文件</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
